L1Trigger/L1THGCalUtilities/plugins/ntuples/HGCalTriggerNtupleHGCClusters.cc

```diff
@@ -62,19 +63,21 @@ initialize(TTree& tree, const edm::ParameterSet& conf, edm::ConsumesCollector&&
   clusters_token_ = collector.consumes<l1t::HGCalClusterBxCollection>(conf.getParameter<edm::InputTag>("Clusters"));
   multiclusters_token_ = collector.consumes<l1t::HGCalMulticlusterBxCollection>(conf.getParameter<edm::InputTag>("Multiclusters"));
 
-  tree.Branch("cl_n", &cl_n_, "cl_n/I");
-  tree.Branch("cl_id", &cl_id_);
-  tree.Branch("cl_mipPt", &cl_mipPt_);
-  tree.Branch("cl_pt", &cl_pt_);
-  tree.Branch("cl_energy", &cl_energy_);
-  tree.Branch("cl_eta", &cl_eta_);
-  tree.Branch("cl_phi", &cl_phi_);
-  tree.Branch("cl_layer", &cl_layer_);
-  tree.Branch("cl_subdet", &cl_subdet_);
-  tree.Branch("cl_cells_n", &cl_cells_n_);
-  tree.Branch("cl_cells_id", &cl_cells_id_);
-  tree.Branch("cl_multicluster_id", &cl_multicluster_id_);
-  tree.Branch("cl_multicluster_pt", &cl_multicluster_pt_);
+  prefix_ = conf.getUntrackedParameter<std::string>("Prefix", "cl");
```

Comment:
Would be better in the constructor

Reply:
But the idea here is to be able to set the prefix from the configuration ParameterSet, and we don't get access to the conf until initialize()

Reply:
`conf` is also passed to the constructor. There are already some parameters configured there.

Reply:
Indeed, I had somehow missed that.
Actually, looking at this again, there is absolutely no need for prefix to be a class member. All we need is a local variable in the initialize function.